simulator/logger.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+class Logger(object):
+	def __init__(self):
+        # Create the Logger
+		self.logger = logging.getLogger("cube")
+		self.logger.setLevel(logging.DEBUG)
+
+		#clear our log
+		open('logging.log', 'w').close()
+ 
+		# Create the Handler for logging data to a file
+		logger_handler = logging.FileHandler('logging.log')

```

Comment:
Can the log file location be something that we set in the config file? (Obviously won't work now with circular imports, but if you implement my other suggestions, it should.)